saturn-datepicker/src/datepicker/calendar.ts

```diff
@@ -204,37 +212,38 @@ export class SatCalendarFooter<D> {
 })
 export class SatCalendar<D> implements AfterContentInit, AfterViewChecked, OnDestroy, OnChanges {
 
-    /** Beginning of date range. */
-    @Input()
-    get beginDate(): D | null { return this._beginDate; }
-    set beginDate(value: D | null) {
-        this._beginDate = this._getValidDateOrNull(this._dateAdapter.deserialize(value));
-    }
-    private _beginDate: D | null;
-
-    /** Date range end. */
-    @Input()
-    get endDate(): D | null { return this._endDate; }
-    set endDate(value: D | null) {
-        this._endDate = this._getValidDateOrNull(this._dateAdapter.deserialize(value));
-    }
-    private _endDate: D | null;
+  /** Beginning of date range. */
+  @Input()
+  get beginDate() {
+    return this._beginDate;
```

Comment:
You have lost the type

Reply:
Yes, because the type now can either be Date | number | null